Stripe/STPBancontactSourceInfoDataSource.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithSourceParams:(STPSourceParams *)sourceParams {
     return self;
 }
 
-- (STPSourceParams *)completedSourceParams {
+- (STPSourceParams *)completeSourceParams {
```

Comment:
Should we build this validation into STPSourceParams somehow? Or have a separate validator class as well? Instead of having it in this VC-specific data source type?